docs/changes.rst

```diff
@@ -6,6 +6,7 @@ Not yet released.
 **New features**
 
 * :ref:`check-rst-references` check to validate reStructuredText references.
+* New `bbcode-text` flag to activate BBCode check was added, see :ref:`custom-checks`.
```

Comment:
Can you please describe this in the compatibility section below so that it is clear that users relying on this check need to add this flag manually now.